components/Topbar/Sidebar.tsx

```diff
@@ -120,6 +127,18 @@ export const Sidebar = () => {
                 Submit project
                 <MdPostAdd className="text-2xl" />
               </button>
+              {session === null && (
+                <Button
+                  onClick={handleModal}
+                  isDisabled={false}
+                  className="text-md w-full py-3 px-6 font-semibold md:w-auto md:text-lg"
+                >
```

Comment:
just a change request, could you make this login item in sidebar similar to other sidebar buttons?

Reply:
sure

Reply:
@rohitdasu are you talking about color or the style format?

Reply:
both, it should match with other items.

Reply:
@rupali-codes

Reply:
alright

Reply:
done @rohitdasu

Reply:
thanks @rupali-codes great job